imports/users.js

```diff
@@ -32,5 +32,9 @@ export const userSettings = {
         meetingSeriesList: 'showQuickHelp_meetingSeriesList',
         meetingSeries:     'showQuickHelp_meetingSeries',
         meeting:           'showQuickHelp_meeting'
+    },
+
+    showAddDetail: {
+        topicInfoItemEdit: 'showAddDetail_topicInfoItemEdit'
```

Comment:
see above - remove the extra level of settings: topicInfoItemEdit. We only need "showAddDetail" here...